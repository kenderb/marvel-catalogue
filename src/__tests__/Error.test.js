import { create } from 'react-test-renderer';
import Error from '../components/Error';

test('Error renders correctly', () => {
  const tree = create(<Error />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Error is not a string', () => {
  const tree = create(<Error />).toJSON();
  expect(tree).not.toBe(typeof String);
});
