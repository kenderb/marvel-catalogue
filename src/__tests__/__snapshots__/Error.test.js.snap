// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders correctly 1`] = `
<h1>
  ERROR!!!
</h1>
`;
